docs/src/appendix/cheatcodes/start_mock_call.md

```diff
@@ -44,12 +55,73 @@ We can use `start_mock_call` in a test to change the data returned by `get_balan
 #[test]
 fn test_mock_call() {
     // ...
-    
+
     let mock_ret_data = 421;
     start_mock_call(contract_address, 'get_balance', mock_ret_data);
 
     dispatcher.set_balance(13);
     let balance = dispatcher.get_balance();
     assert(balance == 421, 'Wrong balance'); // this assert passes
 }
-```
\ No newline at end of file
+```
+
+## Mocking non-existent functions
+
+It is also possible to simulate contract having a specific method by mocking a non-existent selector.
+
+Let's assume we have defined a interface to use a dispatcher, but this interface contains a function that is not
+actually defined in actually deployed contract:
+
+```rust
+// ...
+// Assume we define an interface like that, but it is incorrect.
```

Comment:
```suggestion
// Assume we define an interface like that, but it is extended.
```
Why would anyone define an incorrect interface?

Reply:
I guess not, I meant that it contains a function that was not actually implemented in the main contract. I'll reword it so it explains it better.